tensorboard/uploader/uploader.py

```diff
@@ -330,12 +330,16 @@ def __init__(
         self._scalar_request_sender = _ScalarBatchedRequestSender(
             experiment_id, api, rpc_rate_limiter,
         )
+        self._tensor_request_sender = _TensorBatchedRequestSender(
+            experiment_id,
+            api,
+            # BDTODO: ASK REVIEWERS: Should we use a different rate limiter?
```

Comment:
Question for reviewers: Should I reuse the rate limiter for _ScalarBatchedRequestSender? I think I should.

Reply:
On server side, we can set those limits per rpc request, so we should either have that ability here, or we're stuck setting the limit to the lowest value of the rate limiter.  Makes me think long term, we probably don't want to have set the limit both places.  Server only with retry/backoff logic in the uploader would probably be nicer, but out of scope for this PR.

I'm personally OK with re-using the scalar batch, but I think its shortcut we'll need to remove in the future.

Reply:
I agree that there are open questions about the best way to do this later, worth leaving a TODO about, but for now reusing the scalars rate limiter is fine.